cognee/api/v1/permissions/routers/get_permissions_router.py

```
@@ -2,45 +1,63 @@
+        if not permission_entity:
+            stmt = insert(Permission).values(name=permission)
+            await db.session.execute(stmt)
+            permission_entity = (
+                await db.session.execute(select(Permission).where(Permission.name == permission))).scalars().first()
 
-        group.permissions.append(permission)
+        try:
+            # add permission to group
+            await db.session.execute(
+                insert(GroupPermission).values(group_id=group.id, permission_id=permission_entity.id))
+        except IntegrityError as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)